Scenarioo/ScenariooTest/ScenariooTest/ScenarioDocuWriterTest.cs

```diff
@@ -34,147 +35,153 @@
 
 namespace ScenariooTest
 {
-    [TestClass]
+    [TestFixture]
     public class ScenarioDocuWriterTest
     {
-        private const string BranchName = "testBranch";
-
+        private const string BranchName = "csharp-writer-unit-tests";
         private const string BuildName = "testBuild";
-
-        private const string UseCaseName = "testCase";
-
+        private const string SerializationUseCase = "Serialization";
         private const string ScenarioName = "testScenario";
-
         private const string DetailsVersionKey = "version";
-
         private const int StepIndex = 1;
         
         private string rootDirectory;
 
         private ScenarioDocuWriter writer;
-
         private ScenarioDocuReader reader;
-
         private ScenarioDocuFiles docuFiles;
 
-        [TestInitialize]
+        [TestFixtureSetUp]
         public void TestInit()
         {
             // Sets outcome directory
-            this.rootDirectory = Path.Combine(Directory.GetCurrentDirectory(), "testoutcome");
-
-            if (!Directory.Exists(this.rootDirectory))
+            rootDirectory = Path.Combine(Directory.GetCurrentDirectory(), "testoutcome");
+            TestCleanUp();
+            
+            if (!Directory.Exists(rootDirectory))
             {
-                Directory.CreateDirectory(this.rootDirectory);
+                Directory.CreateDirectory(rootDirectory);
             }
 
-            this.writer = new ScenarioDocuWriter(
-                this.rootDirectory,
+            writer = new ScenarioDocuWriter(
+                rootDirectory,
                 BranchName,
                 BuildName);
 
-            this.reader = new ScenarioDocuReader(this.rootDirectory);
+            reader = new ScenarioDocuReader(rootDirectory);
 
-            this.docuFiles = new ScenarioDocuFiles(this.rootDirectory);
+            docuFiles = new ScenarioDocuFiles(rootDirectory);
         }
 
-        [TestCleanup]
+        //[TestFixtureTearDown]
```

Comment:
yes, i'll clean it up. the tests used to delete allt he output, but you can actually point the scenarioo-ui directly to the out dir which is quite helpful.

ScenarioDocuReader:
it's possible to read, as long as there are no "Details" on the object. so most of the objects not. i don't like this at all, we already spoke about that... (json).